src/packages/client/src/generation/TSClient/constants.ts

```diff
@@ -61,7 +61,20 @@ const JSDocFields = {
 }
 export const JSDocs: JSDocsType = {
   groupBy: {
-    body: () => `Group By`,
+    body: (ctx) => `Group by ${ctx.singular}.
+@param {${getGroupByArgsName(ctx.model.name)}} args - Group by arguments.
+@example
+// Group by city, order by createdAt, get count
+const result = await prisma.user.groupBy({
+  by: ['city', 'createdAt'],
+  orderBy: {
+    createdAt: true
+  },
+  count: {
+    _all: true
+  },
+})
+`,
```

Comment:
This looks hardcoded @timsuchanek should be dynamic no?

Reply:
I intentionally did a hardcoded example here, as it's otherwise quite hard to auto generate the jsdocs (we need to find the right fields, check if they're even there etc).
So I chose an example that always makes sense.

Reply:
Ok!